docs/source/clusters.rst

```diff
@@ -4,58 +4,55 @@ Clusters
 Background
 ~~~~~~~~~~
 
-StreamAlert will deploy separate infrastructure for each ``cluster`` (or environment) you define.
+StreamAlert can deploy separate infrastructure for each ``cluster`` (or environment) you define.
 
 What constitutes a ``cluster`` is up to you.
 
-Example: You could define ``IT``, ``PCI`` and ``Production`` clusters.
+Example: You could define ``IT``, ``PCI`` or ``Production`` clusters.
 
 Strategy
 --------
 
-Cluster definition is up to you.
-
 Common patterns:
 
 1. Define a single cluster to receive and process data from all of your environments
 2. Define a cluster for each of your environments
 3. Define a cluster for each organization, which may have one or more environments
 
-Which one you choose is largely dependent on your processes, requirements and how your team organizes itself
+Which one you choose is largely dependent on your processes, requirements and how your team organizes itself.
 
-Option \(2\) is encouraged because it provides segmentation for ingestion, analysis and storage, on a per-cluster basis, ensuring that folks only have access to the infrastructure and data they need to get their job done.
+Option \(2\) is encouraged because it provides segmentation for ingestion and analysis, which allows for the best possible performance during processing.
 
 Configuration
 -------------
 
-Each cluster lives in its own ``json`` file in the ``conf/clusters`` directory.
-
-A cluster file contains name, region, modules, and outputs.
+Each cluster file lives in its own ``JSON`` file in the ``conf/clusters`` directory, and contains a name, region, modules, and outputs.
```

Comment:
`outputs` in this sentence is a little misleading here - can we omit or clarify what kind of `outputs` this is referring to?